### ISSUE TITLE
Error by Creating Farm from Example Script "SingleServer"

### ISSUE THREAD

**Issue by XdrXdr77** [PowerShell/xSharePoint#150]
Hello @ all

After testing the Installation of SharePoint 2016 with DSC and some hard days, i went back to test the Installation of SharePoint 2013 with DSC ;)

I´ve installed the PReReq and Binaries successfullly, but now it stuck by creating the Configuration File.

I get the following error:

```
ImportCimAndScriptKeywordsFromModule : Cim deserializer threw an error when deserializing file C:\Program Files\WindowsPowerShell\Modules\xSharePoint\DSCResources\MSFT_xSPWebApplicationAppDomain\MSFT_xSPWebApplicationAppDomain.Schema.mof.
At C:\Windows\system32\WindowsPowerShell\v1.0\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:1972 char:50
+ ... dResource = ImportCimAndScriptKeywordsFromModule -Module $mod -Resour ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], PSInvalidOperationException
    + FullyQualifiedErrorId : System.Management.Automation.PSInvalidOperationException,ImportCimAndScriptKeywordsFromModule
 
ImportCimAndScriptKeywordsFromModule : Syntax error: 
 At line:30, char:10
 Buffer:
] boolean SSL
    [Write^, Embed
At C:\Windows\system32\WindowsPowerShell\v1.0\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:1972 char:50
+ ... dResource = ImportCimAndScriptKeywordsFromModule -Module $mod -Resour ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], CimException
    + FullyQualifiedErrorId : Microsoft.Management.Infrastructure.CimException,ImportCimAndScriptKeywordsFromModule
```

I use the example script for a "Single Server"

I need a tip how to fix this problem. Has anyone the same problem and have a solution for this.

Kind Regards

A.S:

**Comment by XdrXdr77** [comment]
Hello,

i´ve checked the schema.mof and find the following in Line 29.

```
    [Write] boolean SSL
```

The semicolon is missing after the SSL

I think it must be set. Is this correct ?

I´ve got another question.

How is the correct call to build the .mof file for the configuration. I modified the SharePoint.psd1 with my settings but when i call the SharePoint.ps1 example script, i won´t build the .mof file.

Kind Regards

A.S.

**Comment by camiloborges** [comment]
Hello. @BrianFarnhill  I've fixed the issue and created a PR for that. I don't know the answer to why the ps1 isn't producing the mof

**Comment by BrianFarnhill** [comment]
Yea I don't have a good reason off the top of my head about why the MOF isnt being generated - normally it'll spit out a partial MOF and then an error.mof file with more details.

There was a test that was checking the MOF files better which @camiloborges and I broke when we set the MOF scanner up to handle multiple objects in a file that I have fixed in a branch I'm working on (see https://github.com/BrianFarnhill/xSharePoint/tree/bug-fixmoftests for the code) which fixes this and a few other issues. I'll have a good look when I get back from leave next week though.

**Comment by XdrXdr77** [comment]
Hi @ all

i can create the MOF FIle. IT wa smy fault. I didn´t called the .psd1. 

I found an other error in your example Script. I created a Webapplication, but the Databasename was "WSS_Content", but i named it SP_Content_01. So i check the script.

In this section:

```
WebApplications = @(
                @{
                    Name = "Portal"
                    DatabaeName = "SP_Content_01"
                    Url = "http://portal.dev.local"
                    Authentication = "NTLM"
                    Anonymous = $false
                    AppPool = "Portal"
                    AppPoolAccount = "DEV\SVC_SP_ServiceApp"
                    SuperUser = "DEV\SVC_SP_SuperUser"
                    SuperReader = "DEV\SVC_SP_SuperReader"
                    UseHostNamedSiteCollections = $true
```

is a "s" missing for the DatabaseName

Greetz

A.S.

**Comment by BrianFarnhill** [comment]
I'll get that typo fixed up in the sample file

**Comment by XdrXdr77** [comment]
Thx a lot ;)

I´ve on question. I´ve tested the the WMF 4.0 first, but there is a problem with the "PSDSCRunAsCredential" setting. After i´ve installed WMF 5.0 it´s working.
In your requirement is only WMF 4.0 listed. Is this correct or maybe changed to WMF 5.0 ?

Greetz

A.S.

**Comment by BrianFarnhill** [comment]
Thats a PowerShell 5 thing, we do support PowerSehll 4 but it's done differently. Have a read of [this wiki page](https://github.com/PowerShell/xSharePoint/wiki/Remote%20sessions%20and%20the%20InstallAccount%20variable) for the details on what you are seeing.

**Comment by BrianFarnhill** [comment]
The typo that was causing the issue here has been resolved in #157, marking this issue as closed
